lib/tasks/test_security.rake

```diff
@@ -2,28 +2,58 @@ namespace :test do
   namespace :security do
     task :setup # NOOP - Stub for consistent CI testing
 
-    desc "Run Brakeman"
-    task :brakeman do
+    desc "Run Brakeman with the specified report format ('human' or 'json')"
+    task :brakeman, :format do |_, args|
+      format = args.fetch(:format, "human")
+
+      require "vmdb/plugins"
       require "brakeman"
 
       # See all possible options here:
-      #   http://www.rubydoc.info/gems/brakeman/Brakeman#run-class_method
-      tracker = Brakeman.run(
-        :app_path     => ".",
+      #   https://brakemanscanner.org/docs/brakeman_as_a_library/#using-options
+      options = {
+        :app_path     => Rails.root.to_s,
+        :engine_paths => Vmdb::Plugins.paths.values,
         :quiet        => false,
         :print_report => true
-      )
+      }
+      if format == "json"
+        options[:output_files] = [
+          Rails.root.join("log/brakeman.json").to_s,
+          Rails.root.join("log/brakeman.log").to_s
+        ]
+      end
+
+      tracker = Brakeman.run(options)
 
-      # Exit 1 on any warnings so CI can report the project as red.
-      exit tracker.filtered_warnings.empty? ? 0 : 1
+      exit 1 unless tracker.filtered_warnings.empty?
     end
 
-    desc "Run bundler audit"
-    task :bundler_audit do
-      exit $?.exitstatus unless system("bundle-audit check --update --verbose")
+    desc "Run bundler audit with the specified report format ('human' or 'json')"
+    task :bundler_audit, :format do |_, args|
+      format = args.fetch(:format, "human")
+
+      options = [:update, :verbose]
+      if format == "json"
+        options << {
+          :format => "json",
+          :output => Rails.root.join("log/bundle-audit.json").to_s
```

Comment:
@Fryguy should the log file name match the task name?  `bundler-audit.json` vs `bundle-audit.json`

Reply:
I actually have a change locally that does it the other way around - changing the task name to bundle_audit to match the executable name.